ui/components/MeshModelRegistry/MeshModelComponent.js

```diff
@@ -367,10 +436,33 @@ const MeshModelComponent_ = ({
   useEffect(() => {
     fetchData();
   }, [view, page, rowsPerPage, checked, searchText, modelFilters, registrantFilters]);
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await fetch('/api/schema/resource/model', {
```

Comment:
Please don't use fetch API.
We are trying to move away from fetch API and use rtk query. // @captain-Akshay May be you can help @Jougan-0 here.

Reply:
This was used for pattern import I just copied the functions and updated it.

Reply:
I see, the migration to rtk isn't complete, but as we are defining/calling APIs let's try to no use fetch.

Reply:
@captain-Akshay